tests/common/helpers.py

```
@@ -54,8 +54,8 @@ def next_random_bytes(rng: random.Random, n: int = 16):
     return rng.randbytes(n)
 
 
-def next_ecdh_keypair():
-    private_key = ec.generate_private_key(curve=ec.SECP384R1())
+def next_ecdh_keypair(curve=ec.SECP256R1()):
```

Ruff (0.15.10)

[warning] Do not perform function call `ec.SECP256R1` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)